cleanup.go

```diff
@@ -8,16 +8,63 @@ import (
 	"time"
 )
 
+// DefaultTimeout for termination
+var DefaultTimeout = 10 * time.Second
+
 // terminateOptions is a type that holds the options for terminating a container.
 type terminateOptions struct {
-	ctx     context.Context
-	timeout *time.Duration
-	volumes []string
+	ctx         context.Context
+	stopTimeout *time.Duration
+	volumes     []string
 }
 
 // TerminateOption is a type that represents an option for terminating a container.
 type TerminateOption func(*terminateOptions)
 
+// NewTerminateOptions returns a fully initialised TerminateOptions.
+// Defaults: StopTimeout: 10 seconds.
+func NewTerminateOptions(ctx context.Context, opts ...TerminateOption) *terminateOptions {
```

Comment:
suggestion: The return type should be exported, as per my previous suggestion. Just keeping fields private ensures the behaviour is consistent.